drivers/platform/stm32/stm32_spi.c

```diff
@@ -200,6 +200,7 @@ int32_t stm32_spi_init(struct no_os_spi_desc **desc,
 		goto error;
 
 	/* copy settings to device descriptor */
+	spi_desc->parent = param->parent;
```

Comment:
move this line here:
https://github.com/analogdevicesinc/no-OS/blob/master/drivers/api/no_os_spi.c#L60